### PR TITLE
Add storage scale-out recipe.

### PR DESCRIPTION
**Pull request by artek-koltun**
This recipe describes how to hot-plug/hot-unplug 64 virtio-blk devices.

### CODE REVIEW

File: build/storage/recipes/README.md [hunk 1]
```diff
@@ -6,8 +6,10 @@ different machines and demonstrates virtio-blk hot-plug to a running host.
 * [fio](./fio.md) - extends `hot-plug` scenario and describes how to run
 fio traffic by means of `host-target` container through a hot-plugged virtio-blk
 device.
+* [scale-out](./scale-out.md) - a very similar to hot-plug recipe, but describing
```
Comment (intelfisz) [inline build/storage/recipes/README.md:9]: similar to hot-plug recipe but describing
how to hot-plug/hot-unplug 64 virtio-blk devices.

File: build/storage/recipes/scale-out.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+# Scale-out recipe
+
+This recipe describes how to attach 64 virtio-blk disks to a Host
```
Comment (intelfisz) [inline build/storage/recipes/scale-out.md:3]: do we capitalize 'host'?

File: build/storage/recipes/scale-out.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+with IPDK containers. This scenario is very similar to
```
Comment (intelfisz) [inline build/storage/recipes/scale-out.md:4]: stripe 'very'

File: build/storage/recipes/scale-out.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+[hot-plug scenario](hot-plug.md) but attaching 64 virtio-blk devices.
```
Comment (intelfisz) [inline build/storage/recipes/scale-out.md:5]: but 64 virtio-blk devices are attached.

File: build/storage/recipes/scale-out.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+For this recipe, 2 machines required.
```
Comment (intelfisz) [inline build/storage/recipes/scale-out.md:7]: two physical machines are required.

File: build/storage/recipes/scale-out.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+They are referred to as `storage-target-platform` and `ipu-storage-container-platform`.
+The containers running on those platforms are named `storage-target` and
+`ipu-storage-container` respectively.
```
Comment (intelfisz) [inline build/storage/recipes/scale-out.md:10]: ',' before respectively

File: build/storage/recipes/scale-out.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+for ((i=0; i < "64"; i++)) ; do \
+virtio_blks+=("$(create_virtio_blk_without_disk_check <ipu_storage_container_platform_ip> \
+        "${ramdrives[$i]}" ${i} 0 nqn.2016-06.io.spdk:cnode0 \
+        "<storage_target_platform_ip>")") ; \
+done
+```
+
+6. Check if virtio-blk is attached to the vm.
+Open the [vm console](environment_setup.md#vm-console) and run the following command
+to Calculate number of virtio-blk devices attached to vm
```
Comment (intelfisz) [inline build/storage/recipes/scale-out.md:65]: calculate - lower case
attached to the vm.

do we use 'VM' or 'vm' consistently?

File: build/storage/recipes/scale-out.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+7. Perform unplug.
+Run the command below on `cmd-sender` to hot-unplug device
+```
+for ((i=0; i < "64"; i++)) ; do \
+    delete_virtio_blk <ipu_storage_container_platform_ip> "${virtio_blks[$i]}"; \
+done
+```
+
+8. Check there is no virtio-blk device.
```
Comment (intelfisz) [inline build/storage/recipes/scale-out.md:83]: that there is no

**Review by intelfisz — CHANGES_REQUESTED** [review]

File: build/storage/recipes/scale-out.md (hunk 3 above)
Comment (artek-koltun) [reply build/storage/recipes/scale-out.md:4]: Done

File: build/storage/recipes/scale-out.md (hunk 6 above)
Comment (artek-koltun) [reply build/storage/recipes/scale-out.md:10]: Done

File: build/storage/recipes/scale-out.md (hunk 8 above)
Comment (artek-koltun) [reply build/storage/recipes/scale-out.md:83]: Done

File: build/storage/recipes/scale-out.md (hunk 7 above)
Comment (artek-koltun) [reply build/storage/recipes/scale-out.md:65]: Done.
Now, we use vm in lower case in all recipes

File: build/storage/recipes/scale-out.md (hunk 6 above)
Comment (artek-koltun) [reply build/storage/recipes/scale-out.md:10]: Done

File: build/storage/recipes/scale-out.md (hunk 4 above)
Comment (artek-koltun) [reply build/storage/recipes/scale-out.md:5]: Done

File: build/storage/recipes/scale-out.md (hunk 2 above)
Comment (artek-koltun) [reply build/storage/recipes/scale-out.md]: no, it is made lower case in all recipes.

**Review by intelfisz — APPROVED** [review]

**Pull request merged**